tests/unittests/Foundation/NSPredicateTests.m

```diff
@@ -113,20 +112,17 @@ - (instancetype)initWithValue:(NSInteger)value {
 
     NSPredicate* unArchivedPredicate = [NSKeyedUnarchiver unarchiveObjectWithData:data];
     ASSERT_TRUE_MSG(unArchivedPredicate != nil, "FAILED: unArchivedPredicate should be non-null!");
-
-    [predicate release];
 }
 
 TEST(NSPredicate, copy) {
-    NSPredicate* predicate = [[NSPredicate alloc] init];
+    NSPredicate* predicate = [NSPredicate predicateWithFormat:@"1<3"];
```

Comment:
Given this is just to obtain a valid object, this should be fine for the scope of this test.

---
In reply to: [73250728](https://github.com/Microsoft/WinObjC/pull/788#discussion_r73250728) [](ancestors = 73250728)